src/atc/process/etl_parameters.py

```diff
@@ -0,0 +1,2 @@
+class ETLParameters:
+    ...
```

Comment:
what does this do?

Reply:
For now its a placeholder class. In the future - default arguments for etl flows can be set here. 

I.e., we're building a "data quality logger" that can be applied by only setting a logger-parameter to True. Now, we want this parameter to be set across all inheriting flows, we set it here. If we want to disable (set the logger-parameter to False) for a specific flow we just do that in the parameter file (class) for the inheriting flow (parameter object).